apps/myaccount/src/components/overview/overview.tsx

```diff
@@ -168,19 +171,30 @@ export const Overview: FunctionComponent<OverviewPropsInterface> = (
         <Grid className="overview-page">
             <Divider hidden />
             <Grid.Row>
-                {
-                    !enableAlternateWidgetLayout ?
+                {!enableAlternateWidgetLayout
```

Comment:
Possible lint error here. Make sure ESLint is configured.
```suggestion
                {
                    !enableAlternateWidgetLayout
```

Reply:
Addressed with c7c686384b74830761d277e3ff4185cb78111ba2